rootfs/api/models.py

```diff
@@ -1046,7 +1036,7 @@ def save(self, *args, **kwargs):
             config['domains'] = ''
 
         # convert from string to list to work with and filter out empty strings
-        domains = filter(None, config['domains'].split(','))
```

Comment:
is filter slower than iterating through the entire list, or is it gone in python 3.5?